src/App.tsx

```diff
@@ -1,16 +1,77 @@
-import React from 'react';
+import React, { useEffect } from 'react';
```

Comment:
I am afraid that this code is going to be deleted, after your merge since this code could be partially used  in the Login Page.

Reply:
Yeah, the purpose of that code is just to show how graphQL server works with client side.
But still I belive even this temporarily code with simple login logic can save some time while building `Login` component.